lua/apisix/plugins/jwt-auth.lua

```diff
@@ -89,15 +90,19 @@ function _M.check_schema(conf)
 end
 
 
-local function fetch_jwt_token()
-    local args = ngx.req.get_uri_args()
-    if args and args.jwt then
-        return args.jwt
+local function fetch_jwt_token(ctx)
+    local token = core.request.header(ctx, "authorization")
+    if token then
+        local prefix = sub_str(token, 1, 7)
+        if prefix == 'Bearer ' or prefix == 'bearer ' then
+            return sub_str(token, 8)
+        end
+        return token
     end
 
-    local headers = ngx.req.get_headers()
-    if headers.Authorization then
-        return headers.Authorization
+    token = ngx.ctx.api_ctx.var.arg_jwt
```

Comment:
`ngx.ctx. api_ctx` equals to `ctx`. 

So this line should be `token = ctx.var.arg_jwt` which is simpler.

Reply:
ok,changed